pkg/otlp/config_test.go

```diff
@@ -480,6 +495,7 @@ func TestFromAgentConfigDebug(t *testing.T) {
 				TracePort:                5003,
 				MetricsEnabled:           true,
 				TracesEnabled:            true,
+				LogsEnabled:              true,
```

Comment:
Please add a test on DD_OTLP_CONFIG_LOGS_ENABLED to TestFromEnvironmentVariables